samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -523,31 +530,38 @@ public virtual Response StartTranslation(RequestContent content, RequestOptions
             {
                 message.SetProperty("RequestOptionsPerCallPolicyCallback", options.PerCallPolicy);
             }
-            using var scope = _clientDiagnostics.CreateScope("DocumentTranslationClient.StartTranslation");
-            scope.Start();
-            try
+
+            Response BeginOperation()
             {
-                Pipeline.Send(message, options.CancellationToken);
-                if (options.StatusOption == ResponseStatusOption.Default)
+                using var scope = _clientDiagnostics.CreateScope("DocumentTranslationClient.StartTranslation");
+                scope.Start();
+                try
                 {
-                    switch (message.Response.Status)
+                    Pipeline.Send(message, options.CancellationToken);
+                    if (options.StatusOption == ResponseStatusOption.Default)
```

Comment:
Does the operation behave correctly when we create it with non-successful response?

Reply:
Thinking about this more - I'm not even sure what we should do in this case.  I think the most sensible thing we could do is return some `Operation<BinaryData>` where we throw the RequestFailedException when WaitForCompletionAsync is called?

Reply:
Interested in your thoughts here as well @KrzysztofCwalina, does this seem like a reasonable thing to do with these `Operation` methods?  I assume we'd do something similar for Pagable where the exception is throw when you actually try to enumerate the pageable that is returned?

Reply:
I looked at what happens today and what I described above (an operation that will throw a RequestFailedException when you try to wait for it it) is what happens with the code as is today.